src/ui/camera.js

```diff
@@ -192,8 +192,8 @@ class Camera extends Evented {
      * Pans the map by the specified offset.
      *
      * @memberof Map#
-     * @param offset `x` and `y` coordinates by which to pan the map.
-     * @param options Options object.
+     * @param offset The `x` and `y` coordinates by which to pan the map.
+     * @param options An options object describing the destination and animation of the transition. We do not recommend using `options.offset` since this value will override the value of the `offset` parameter.
```

Comment:
`CameraOptions` is used by a bunch of different apis and `offset` makes sense for some of them (`jumpTo`, `easeTo`) so I think we'll need to either:
- change it to something like "We do not recommend using `options.offset` **with panBy**..."
- move this disclaimer to only to `panBy`'s documentation

Reply:
> move this disclaimer to only to panBy's documentation

🤔 Hmmm, I thought this was the documentation for `panBy`?  https://docs.mapbox.com/mapbox-gl-js/api/map/#map#panby does seem to suggest that the docs would be in the `map.js` file, but the `camera.js` file is the only place I found it, and references to `map.panBy` all seem to lead here (ex [this entry in the changelog](https://github.com/mapbox/mapbox-gl-js/blob/main/CHANGELOG.md#-bug-fixes-4)).

Reply:
Woops, sorry. You're completely right! I'm not sure how I misread this

Reply:
Thanks @ansis! I appreciate your help!